pootle/core/initdb.py

```diff
@@ -244,15 +253,16 @@ def create_default_projects(self):
         """
         en = self.require_english()
 
+        po = Format.objects.get(name="po")
```

Comment:
Nit: move this down to before `tutorial, created = self._create_object(Project, **criteria)`. This way changes are closer and this method resembles like previous one.